### PR TITLE
fill out readme

### PR DESCRIPTION
**Pull request by autopulated**
cc @iriark01

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+
+### Using in yotta target descriptions
+
+To use valinor to add debug support to a yotta target description add this to
+your target.json file (replacing K64F with the project_generator target ID for
+the chip or board on your target):
+
+```json
+    "scripts":{
+        "debug": ["valinor", "--target", "K64F", "$program"]
```
Comment (0xc0170) [inline readme.md]: Can we change K64F to frdm-k64f? :-)

Comment (autopulated) [inline readme.md]: Sure — that'll work already, right?

Comment (0xc0170) [inline readme.md]: Yes, it should and I should have changed it, time to test and do :-)

**Comment by autopulated** [comment]
OK. I'm going to merge now as the tests failing is nothing to do with these changes.

**Pull request merged**